UT4MasterServer/Services/MatchmakingService.cs

```diff
@@ -62,8 +70,8 @@ public async Task<List<GameServer>> ListAsync(GameServerFilter inputFilter)
 		// Build BsonDocument representing Find filter
 		var doc = new BsonDocument();
 
-		// include GameServers that have started
-		doc.Add(new BsonElement(nameof(GameServer.Started), true));
+		//// include GameServers that have started
```

Comment:
```suggestion
		// include GameServers that have started
```

Reply:
There are 4 slashes because i re-commented this and commented one line under this. It's kind of indicating that this was a comment to the code in commented next line.